alissa_API_tools.py

```diff
@@ -68,9 +68,10 @@ def alissa_upload(accession, sex, alissa_folder, vcf_path, output_folder, size,
               help='Path to a folder where the VCF will be written if the input VCF is larger than the size argument. In that case, files will be loaded to Alissa from that folder. Default: "/tmp".')
 @click.option('-i', '--production-instance', type=click.Choice(['production', 'test']) , default="test",
              help='What Alissa instance should be used. Default: "test".') #TODO invert the logic once testing is finished.
+@click.option('-ref', '--reference', help='Reference genome (hg19 or hg38). Default: hg19', default = 'hg19')
```

Comment:
Change this to ` '-ref', help= '', type=click.Choice(['hg19', 'hg38'], case_sensitive=False, default='hg19'))` to ensure no other weird inputs are accepted.

Reply:
solved in 58a48a3